jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemPresenter.java

```diff
@@ -102,6 +103,23 @@ public void signalParameterAddedOrRemoved() {
         fireChangeEvent();
     }
 
+    public void openEditModal(final String name,final String value) {
+        ItemObjectModel itemObjectModel = new ItemObjectModel();
+        itemObjectModel.setParameters(model.getParameters());
+        itemObjectModel.setResolver(model.getResolver());
+        
+        BiConsumer<String,String> removeConsumer = (n,v) -> {
+            super.remove();
+        };
+        
+        BiConsumer<String,String> editConsumer = removeConsumer.andThen( (n,v) ->{
+            itemObjectModel.setName(n);
+            itemObjectModel.setValue(v);
+            this.getSectionListPresenter().add(itemObjectModel);
+            fireChangeEvent();
+        });
+        this.getSectionListPresenter().showDoubleValueEditModal(name,value, editConsumer);
```

Comment:
yes,I just want let Section Presenter reuse the valuleModal.
If it is ok, I will change all call new Modal of Section Presenter.

Reply:
you don't necessarily need to reuse the same instance, you can inject a different one for the edit operation, which I think would make sense and simply a lot the changes needed.

Reply:
ok, I will refactor all code.

Reply:
@cristianonicolai Today I want refactor it, I find the way is not make sense that inject valueModal in to every Item Presenter. Because , valueModal need setup itself.
NameObjectItemPresenter have many subclass , NameObjectItemPresenter can not set it by itself.

Reply:
@cristianonicolai  For inject value modal in to NameObjectItemPresenter(Other Item Presenter is same) , we need refactor Section Presenter and NameObjectItemPresenter, like this: https://gist.github.com/bxf12315/7338a7950fba23ecf76066c66285ac59#file-deploymentsworkitemhandlerspresenter-L83
and https://gist.github.com/bxf12315/7338a7950fba23ecf76066c66285ac59#file-namedobjectitempresenter-L56.
In Section Presenter , we need setup value modal twice.